.github/workflows/kubernetes_test.yaml

```diff
@@ -4,33 +4,34 @@ on:
   pull_request:
     paths:
       - ".github/workflows/kubernetes_test.yaml"
-      - "tests/**"
       - "tests_deployment/**"
       - "tests_e2e/**"
-      - "scripts/**"
-      - "nebari/**"
-      - "pyproject.toml"
-      - "pytest.ini"
+      - "nebari/stages/**"
+      - "nebari/template/stages/**"
+
```

Comment:
Why don't we test e.g. nebari/cli/** or nebari/*.py anymore?  Seems like it'd be good to run the kubernetes tests if we make changes to the cli.

Reply:
I am divided in that matter, some commands I definitely agree should be tested within a running instance (e.g. keycloak), but at the end of the day, the CLI is just a wrapper for deployment, which I think we could get around by writing proper tests just for it and keep the kubernetes default trigger run just for e2e. 

As a second thought, I can add a way for triggering this action by adding a label to the PR or a comment, so that it can be manually triggered when we would like a e2e but the changes were done to a untracked location